### ISSUE TITLE
grid tweaks

### ISSUE THREAD

**Issue by davidperrenoud** [fritzing/fritzing-app#1944]
_From [irasc...@gmail.com](https://code.google.com/u/104729248032245122687/) on April 11, 2012 02:05:03_

From André
- Use continuous low-contrast line instead of stippled line
- Give it a shortcut (maybe Ctrl+G?)
- Give every 10th line a little more contrast
- Move the "grid size" setting back to the view menu
- Seems like it causes some new update bugs? For me, when I now delete a part, it doesn't immediately disappear.

_Original issue: http://code.google.com/p/fritzing/issues/detail?id=1944_

**Comment by failiz** [comment]
Thinks that are still not implemented (from first post):

- Give it a shortcut (maybe Ctrl+G?)
- Give every 10th line a little more contrast

Remove imported label

**Comment by KjellMorgenstern** [comment]
I don't think this is worth a distinct shortcut. Alt-V and arrow keys can navigate you there if you really want this setting via a keyboard.

- [ ] Add option to give every 10th line a little more contrast

**Comment by failiz** [comment]
I think that a shortcut to set the grid size would be useful. When you are routing PCBs is very handy: use a rough grid (e.g. 0.1inches) to place and move components and a small one (e.g. 10mils) for routing tracks. Another option would be to allow to have several grid sizes in the settings (rough, normal, fine) and have shortcuts to change between them.
